server/__init__.py

```diff
@@ -126,15 +126,33 @@
     }
 ]
 # Dict of production models, key: model name, value: tuple (clf, generator_fn, predictor_fn).
+# Changed to just load the defualt model
 production_models = {}
 for model_config in MODEL_CONFIGS:
-    print("Loading: " + model_config["name"])
-    clf = joblib.load(model_config["path"])
+    print("Putting in dictionary: " + model_config["name"])
     gen = model_config["generator"]
     pred = model_config["predictor"]
-    production_models[model_config["name"]] = (clf, gen, pred)
+    if model_config["name"] == DEFAULT_MODEL:
+        print("Loading: " + model_config["name"])
+        clf = joblib.load(model_config["path"])
+        production_models[model_config["name"]] = (clf,gen, pred)
+    else:
+        production_models[model_config["name"]] = (None, gen, pred)
 print("\tDone in {:.1f} secs".format(time.time() - start_time))
 
+#Load the called model function
+
+def load_model(name):
+    print("Loading: " + model_config["name"])
```

Comment:
This will result in an error:

```
  File "server/__init__.py", line 146, in load_model
    print("Loading: " + model_config["name"])
UnboundLocalError: local variable 'model_config' referenced before assignment
```

Use `name` instead.